Assets/MRTK/SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -245,15 +245,15 @@ protected override void OnDisable()
 
         private Vector2 currentInputPosition = Vector2.zero;
 
-        protected bool isTeleportRequestActive = false;
+        private bool isTeleportRequestActive = false;
 
         private bool lateRegisterTeleport = true;
 
         private bool canTeleport = false;
 
         private bool canMove = false;
 
-        protected Gradient GetLineGradient(TeleportSurfaceResult targetResult)
+        private Gradient GetLineGradient(TeleportSurfaceResult targetResult)
```

Comment:
Two more instances of visibility being lowered on scripts that might be overridden by consumers of MRTK

Reply:
> Two more instances of visibility being lowered on scripts that might be overridden by consumers of MRTK

So, what sould i do?

Reply:
They should be left as protected to not lower the visibility of these variables

Reply:
Alright, understood.